trappy/ftrace.py

```diff
@@ -75,15 +75,10 @@ def __init__(self, name="", normalize_time=True, scope="all",
             trace_class = class_def()
             setattr(self, attr, trace_class)
             self.trace_classes.append(trace_class)
-
-        self.__parse_trace_file(self.trace_path, window, abs_window)
-        if self.needs_raw_parsing and (self.trace_path_raw is not None):
-            self.__parse_trace_file(self.trace_path_raw, window, abs_window,
-                                    raw=True)
-        self.finalize_objects()
-
-        if normalize_time:
-            self.normalize_time()
+        # save parameters to complete init later
+        self.normalize_time_param = normalize_time
```

Comment:
I think I can't make it completely private, as it is in a base class and called from derived classes, but I'll change it around a bit and fix up the tests which refer directly to it.